### PR TITLE
add: alert + avatar components + utility styles & assets

### PR DESCRIPTION
**Pull request by nayyyhaa**
**Made the following changes:**
- Alert component: default, solid, transition
- Avatar component: image, initials, icon avatar
- Utility classes like: flex, margins, padding, grid, etc
- assets

**current folder structure:**
```
inNotion-UI-library
               |______assets/ imgs
               |______styles
                            |__components-styles
                                          |______alert.css
                                          |______avatar.css
                            |__utils
                                    |______container/flex/grid/utilities/width/gutter/lists/font/button.css
                            |__main.css
               |______app.js
               |______index.html
               |______in-notion.css
               |______README.md
```

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+@import url("utils/gutter.css");
+@import url("utils/width.css");
+
+/* COMPONENTS */
+
+@import url("components-styles/avatar.css");
+@import url("components-styles/alert.css");
+
+:root {
+  --primary-color-xxlight: #f1f0ff;
```
Comment (hsnice16) [inline styles/main.css]: To name variables, use this notation `--COLOR-PRIMARY-XXLIGHT`. it will help to notice that it is a variable.

**Review by hsnice16 — CHANGES_REQUESTED** [review]

File: styles/main.css (hunk 1 above)
Comment (hsnice16) [reply styles/main.css]: And UI looks amazing, Nice work 👍

**Comment by Dev-AkshatGupta** [comment]
Every thing looks great to me but I didn't get the DOM manipulations you want to implement are where they are implemented . Because I was unable to get any kind of functionality.

File: styles/main.css (hunk 1 above)
Comment (nayyyhaa) [reply styles/main.css]: Thanks a lot for reviewing! I've made the following changes for naming CSS variables.

**Comment by nayyyhaa** [comment]
> Every thing looks great to me but I didn't get the DOM manipulations you want to implement are where they are implemented . Because I was unable to get any kind of functionality.

Hey @Dev-AkshatGupta, thanks for reviewing! For the current component (avatar + alert) no DOM manipulation is done. Only for transitional alert, I've added the functionalities in the `app.js` file.

**Comment by Dev-AkshatGupta** [comment]
> > Every thing looks great to me but I didn't get the DOM manipulations you want to implement are where they are implemented . Because I was unable to get any kind of functionality.
> 
> Hey @Dev-AkshatGupta, thanks for reviewing! For the current component (avatar + alert) no DOM manipulation is done. Only for transitional alert, I've added the functionalities in the `app.js` file.

Ok great then  I saw it and I tested the components that's why asked you of out of curiosity keep on doing great work. :)

**Comment by nayyyhaa** [comment]
> > > Every thing looks great to me but I didn't get the DOM manipulations you want to implement are where they are implemented . Because I was unable to get any kind of functionality.
> > 
> > 
> > Hey @Dev-AkshatGupta, thanks for reviewing! For the current component (avatar + alert) no DOM manipulation is done. Only for transitional alert, I've added the functionalities in the `app.js` file.
> 
> Ok great then I saw it and I tested the components that's why asked you of out of curiosity keep on doing great work. :)

Yay, thanks! 👯

**Pull request merged**